### PR TITLE
feat: Add Apify integration

### PR DESCRIPTION
**Pull request by jirimoravcik**
JS version of https://github.com/hwchase17/langchain/pull/2201

If you have any suggestions, feel free to comment on this PR.
Also if there are some issues concerning Apify integrations in langchain, feel free to contact me.

### CODE REVIEW

File: langchain/src/tools/apify.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { Document } from "../document.js";
```
Comment (jacoblee93) [inline langchain/src/tools/apify.ts:1]: Tools have a specific meaning in LangChain - this is in the wrong place. Can we bundle it into the document loader and run `callActor` when `load`'ing documents?

**Review by jacoblee93 — CHANGES_REQUESTED** [review]

File: langchain/src/tools/apify.ts (hunk 1 above)
Comment (jirimoravcik) [reply langchain/src/tools/apify.ts:1]: Thanks for the review.
In the Python version, it's in the utilities. https://github.com/hwchase17/langchain/blob/master/langchain/utilities/apify.py
Not sure if there's an equivalent of it in the JS version?

Comment (jacoblee93) [reply langchain/src/tools/apify.ts:1]: In that case I'd prefer to put this as a `callActor` method in the `document_loader` class. I can make the change today and polish everything up if that's ok with you?

Comment (jacoblee93) [reply langchain/src/tools/apify.ts:1]: The other reason is so that we can give everything that requires Apify one entrypoint.

Comment (jancurn) [reply langchain/src/tools/apify.ts:1]: Thank you @jacoblee93 If you prefer so, feel free to update it that way.

**Comment by jacoblee93** [comment]
Extended and merged here: #1271 

Thanks for your patience!

**Pull request merged**